### ISSUE TITLE
Improvements to method signatures

### ISSUE THREAD

**Issue by FreHu** [andau/abap-code-insight#1]
- ABAP signatures can get rather long. The code mining text does not wrap, even if word wrap is enabled.
![eclipse_2019-01-17_12-07-22](https://user-images.githubusercontent.com/5097067/51314824-0203fe00-1a51-11e9-8d86-96e8f39dc5b3.png)
- Different colors or icons for static/instance, importing/exporting/changing/returning, it would be nice if they matched the ones in element info:
![2019-01-17_12-15-11](https://user-images.githubusercontent.com/5097067/51315063-979f8d80-1a51-11e9-873a-19652afe57cf.png)

**Comment by andau** [comment]
In the latest version I changed the signature format - now it should be more meaningful and the important thing is, its shorter. You have posted a good example, maybe long methods should be shorted by skipping class names of parameter types - have to think about this 

Different colors is currently not possible in eclipse code minings - to add a icon would be possible but I am not sure if this improves the visibility.

**Comment by christianguenter2** [comment]
@andau do you think it's possible to use a new line for each parameter? Imho that would improve readability a lot.

**Comment by FreHu** [comment]
I agree with parameter per line, it is closer to how abap method definitions are written.

Can the text be made selectable? Links to the parameter types would be great to have as well.

**Comment by andau** [comment]
Selectable is not possible currently as far as I know. Links would be possible but the the parameters would be separated with the | symbol as the link can only be set for the whole code insight
